src/Process/PlanarTileProcessing.js

```diff
@@ -42,11 +42,14 @@ export function planarSubdivisionControl(maxLevel, maxDeltaElevationLevel) {
         // Prevent to subdivise the node if the current elevation level
         // we must avoid a tile, with level 20, inherits a level 3 elevation texture.
         // The induced geometric error is much too large and distorts the SSE
-        const currentElevationLevel = node.material.getElevationLayerLevel();
-        if (node.level < context.maxElevationLevel + maxDeltaElevationLevel &&
-            currentElevationLevel >= 0 &&
-            (node.level - currentElevationLevel) >= maxDeltaElevationLevel) {
-            return false;
+        const currentTexture = node.material.textures[0][0];
```

Comment:
this part of this process is identical to the globe processing, it should be encapsulated in one place. Duplicate code is harder and longer to maintain

Reply:
> Duplicate code is harder and longer to maintain

This is not an absolute truth, and here I disagree. This is a 5 line copy-paste so I don't think it hurts to keep them separate.

If you really want to improve code sharing between part of itowns I opened a PR sometimes ago: #674

Reply:
> Duplicate code is harder and longer to maintain

Abusive factorization too, by introducing another layer of abstraction. 

> the duplicate code is technical debt

The program without technical debt is the one with 0 codes. I agree duplicate code has a cost, but introducing another function has one too. I call it more or less even here. But as it is not the scope of this PR and as this snippet is quite short, maybe you can let this one go?

Reply:
This function can be factorized:
  * as an OGC conversion tool. there is WMTS_WGS84Parent. Why not have the opposite function?
  * it's an identified conversion tool, so no code duplication

Reply:
Actually this code is a hack to workaround a bug in the SSE logic (= computing the geometric error using a sphere - except that a sphere is a poor approximation for a square tile).

So I'd rather get rid of it, instead of transforming it in a API that would be used in other places of iTowns.